tests/testcases/hub/execute_hub_operators.py

```diff
@@ -152,31 +192,42 @@ def execute_ops(op_clone_list):
         # extract python code from op readme
         test_case_name = extract_python_code_from_op_readme(op_single_path, op_name_single)
         if not os.path.exists(test_case_name):
-            logger.error("Fail to run readme for %d: operator %s" % (op_id, op_address))
+            logger.error("Fail to extracted readme for %d: operator %s" % (op_id, op_address))
             continue
         op_requirement = op_single_path + "/requirements.txt"
         logger.debug("Install requirements for %s" % op_single_path)
         os.system(f"pip install -r {op_requirement}")
+        # extract all models of each op from yaml
+        logger.debug("Load all models of %s" % op_address)
+        all_models = get_all_models_of_op_from_yaml(op_address)
+        all_models = list(chain.from_iterable(all_models))
         # execute python code from op readme
-        logger.debug("Running extracted readme for %s" % op_single_path)
-        test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py"
-        res = os.system(f"python3 {test_case_name_backup} >> {log_path}")
-        if 0 != res:
-            logger.error("Fail to run readme for %d: operator %s" % (op_id, op_address))
+        is_succeed = True
+        for model in all_models:
+            model_name = model.rsplit('\'', 2)[1]
+            modify_specific_model(test_case_name, model_name)
```

Comment:
I think here should be modify_specific_model(test_case_name_backup, model_name). And move this line after "test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py""

Reply:
> I think here should be modify_specific_model(test_case_name_backup, model_name). And move this line after "test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py""

Done.